APIComparer.Core/TypeDiff.cs

```diff
@@ -55,6 +55,10 @@ public IEnumerable<MatchingMember<FieldDefinition>> PublicFieldsObsoleted()
             return MatchingFields.Where(x => x.Left.IsPublic && x.Right.IsPublic && !x.Left.HasObsoleteAttribute() && x.Right.HasObsoleteAttribute());
         }
 
-    
+        public IEnumerable<MatchingMember<FieldDefinition>> EnumFieldsWithChangedValue()
+        {
+            //TODO: we should actually be comparing based on the enum's underlying type
```

Comment:
Actually no, the comment can be removed since I'm using object.Equals which will call the correct overload based on the underlying type of the enum. I can send a new pull request with the comment line removed if you prefer.